Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.yml

```diff
@@ -0,0 +1,1824 @@
+      type: String
+  - arguments:
+    - default: false
+      description: |-
+        The limit for the number of events listed per page.
+        Default is '50'.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: The number of results per page to display.
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        The page number to view. Each page contains page_size values. Must be used along with page_size.
+        Default is '1'.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: "Specify a search query as a Lucene query string.\nExample: query=\"first_seen: [2022-10-01T07:00:58.030Z  TO 2022-10-21T06:41:54.452Z]\" \n\nNote: For more details refer to Symantec EDR (On-Prem) API document https://apidocs.securitycloud.symantec.com/#"
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: |-
+        Specify a filters option in lieu of “query”. These filters will improve query performance.
+      isArray: false
+      name: search_object
+      predefined:
+      - device_uid
+      - sha256
+      required: false
+      secret: false
+    - default: false
+      description: Specify a search value. Supports a comma-separated query with multiple search values.
+      isArray: false
+      name: search_value
+      required: false
+      secret: false
+    deprecated: false
+    description: List of domain and file association.
+    execution: false
+    name: symantec-edr-endpoint-file-association-list
+    outputs:
+    - contextPath: SymantecEDR.EndpointFileAssociation.device_ip
+      description: 'The IPv6 or IPv4 address of the endpoint when this association was last updated. Example: 127.0.0.1.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.device_name
+      description: 'The host name or, if unavailable, the IP address of the endpoint when this association was last updated. Example: 170915-000020.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.device_uid
+      description: 'Unique ID of the endpoint that has this file. Example: 04cfc04b-5c7a-4aa8-b95b-79be23f768f4.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.frst_seen
+      description: 'The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted into the creation of this association. Example: 2018-02-04T09:00:00.577Z.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.folder
+      description: 'The folder where the file resides. This attribute does not include the name of the file. Example:c:\windows\system32\.'
+      type: String
+    - contextPath: SymantecEDR.EndpointFileAssociation.last_seen
```

Comment:
Why is this description the same as the one for first_seen?  It should be different,